features/import_survey.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Manage Questions
```

Comment:
Super minor nit, but the feature name should be something like "Import Spreadsheet". Probably a copy/paste error.

Reply:
It was a copy paste error, I just pushed the fix as well as some style updates cherrypicked from @mrnosal